tests/suites/1_stateful/00_stage/00_0015_unload_output.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# most features are tested in sqllogic tests with diff type of external stages
+# this test is mainly to test internal stage and user stage (and paths) is parsed and used correctly, the file content is not important
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+export MYSQL="mysql -uroot --host 127.0.0.1 --port 3307  -s"
```

Comment:
BENDSQL_CLIENT?

Reply:
use mysql for test for now because bendsql donot show output of copy now. https://github.com/datafuselabs/bendsql/issues/337